packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTableSet.types.tsx

```diff
@@ -0,0 +1,102 @@
+import { InterfacePropertyType, PropertyType } from '../../utilities/parser/index';
+
+export interface IApiReferencesTableSetProps extends React.HTMLAttributes<HTMLElement> {
+  /**
+   * A json object to populate the table
+   */
+  jsonDocs?: IPageJson;
+}
+
+export interface ITokenJson {
+  text: string;
+  hyperlinkedPage?: string;
+}
+
+export interface ITableRowJson {
+  name: string;
+  typeTokens: ITokenJson[];
+  defaultValue?: string;
+  descriptionHtml: string;
+  deprecated: boolean;
+  deprecatedMessage?: string;
+  kind?: 'Method' | 'Property';
+}
+
+export interface IEnumTableRowJson {
+  name: string;
+  descriptionHtml: string;
+  value: string;
+}
+
+export interface ITableJson {
+  kind: 'interface' | 'enum' | 'class';
+  name: string;
+  extendsTokens: ITokenJson[];
+  descriptionHtml: string;
+  members: ITableRowJson[] | IEnumTableRowJson[];
+}
+
+export interface IPageJson {
+  tables: ITableJson[];
+  name: string;
+}
+
+export interface IApiProperty {
+  propertyName: string;
+  extends?: string;
+  extendsTokens?: ILinkToken[];
+  description?: string;
+  title: string;
+  propertyType: PropertyType;
+  property: IApiInterfaceProperty[] | IApiEnumProperty[];
+  methods?: IMethod[];
+}
+
+/**
+ * Used to keep track of where the page will live on the site
+ */
+export enum PageKind {
+  References = 'References',
+  Components = 'Components'
+}
+
+export interface ILinkToken {
+  text: string;
+  hyperlinkedPage?: string;
+  pageKind?: PageKind;
+}
+
+export interface IApiInterfaceProperty {
+  name: string;
+  typeTokens: ILinkToken[];
+  defaultValue: string;
+  description: string;
+  interfacePropertyType?: InterfacePropertyType;
+  deprecatedMessage?: string;
+  deprecated?: boolean;
+}
+
+// export enum InterfacePropertyType {
+//   required = 0,
+//   optional = 1,
```

Comment:
Do you intend to remove the commented enum and another one on lines 98-102 below?

Reply:
So I have these in here because we have the same enums declared in /utilities/parser/interfaces.ts, which is currently being used in the old PropertiesTable/PropertiesTableSet. I thought it might be easier to keep this in here so I can remember which ones to keep in interfaces.ts when we remove PropertiesTable/PropertiesTableSet eventually? If it's messy though I can just remove it. :)

Reply:
Maybe leave them but also leave a comment why you have them here.